tls/ttls.h

```diff
@@ -37,6 +37,7 @@
 #include "x509_crl.h"
 #include "dhm.h"
 #include "ecdh.h"
+#include "fw/http_types.h"
```

Comment:
Tempesta TLS is going to be moved to upsream and it must not know about the HTTP layer.